ontology/ontoavida.md

```diff
@@ -30,3 +30,12 @@ tags:
 tracker: https://gitlab.com/fortunalab/ontoavida/-/issues
 activity_status: active
 ---
+
+# OntoAvida
+
+<img  src="https://fortunalab.org/images/alife_bacteria.jpg" height="150px" align="left"/>The **Ontology for Avida ([OntoAvida](https://gitlab.com/fortunalab/ontoavida))** project aims to develop an integrated vocabulary for the description of [Avida](https://github.com/devosoft/avida), the most widely used computational approach for performing experimental evolution. The lack of a clearly defined vocabulary makes biologists feel reluctant to embrace the field of digital evolution. This unique ontology has the potential to change this picture overnight. In addition, OntoAvida will allow researchers to make inference (e.g., on phenotypic plasticity) based on certain rules and constraints, facilitate the reproducibility of the *in silico* evolution experiments reported in the [scientific literature](https://gitlab.com/fortunalab/ontoavida#references), and trace the provenance of the data stored in AvidaDB, a semantic database that stores genomes and transcriptomes of more than a million digital organisms.
```

Comment:
there's a structured metadata field for an image/logo

Reply:
Do you mean I should upload the image to logos folder and update the src url to point to it? Thanks

Reply:
It's fine to permalink. See what UBERON did in https://github.com/OBOFoundry/OBOFoundry.github.io/blob/df988b102661c0d6e62fdeaa8e09da07eef78f39/ontology/uberon.md?plain=1#L42

Reply:
Ok I see, but we don't want to use the image as a logo, the image we use is not a logo, we would like to use it as a part of the description.